runtime/doc/options.txt

```diff
@@ -6760,7 +6760,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	determined by the last focused window in the tabpage.  Highlights of
 	the popupmenu are determined by the current window.  Highlights in the
 	message area are not overridable.  Example for overriding the
-	backgrond color: >
```

Comment:
Also `builting` on line 6754 :P

Reply:
fixed in https://github.com/neovim/neovim/pull/7364